### PR TITLE
chore: add syntax highlighting

### PR DESCRIPTION
**Pull request by brnhensley**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Review by ghost — DISMISSED** [review]

**Comment by brnhensley** [comment]
Hey @ally-sassman, I didn't see your approval and I added one more commit to the PR after the fact. Same as the first two, just some syntax highlighting

**Comment by ally-sassman** [comment]
@brnhensley All good, I'll wait for the Gatsby build to complete and then I'll review/merge this PR if you're done.

**Pull request merged**
